bagel/cli.py

```diff
@@ -35,8 +41,8 @@ def pheno(
     name: str = typer.Option(
         ...,
         help="A descriptive name for the dataset the input belongs to. "
-        "This name is expected to match the name field in the BIDS "
-        "dataset_description.json file.",
+        "This name is expected to match the name field in the BIDS dataset_description.json file. "
+        'Should be enclosed in quotes (" ").',
```

Comment:
How about an actual example here? 
```suggestion
        'Should be enclosed in quotes, e.g.: --name "my dataset name".',
```
And maybe rich has something nice to make that part highlighted?